src/main/java/uk/gov/companieshouse/lookup/config/MessageConfig.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.companieshouse.lookup.config;
+
+import java.util.Locale;
+
+import org.springframework.context.MessageSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.springframework.web.servlet.LocaleResolver;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.i18n.LocaleChangeInterceptor;
+import org.springframework.web.servlet.i18n.SessionLocaleResolver;
+
+@Configuration
+public class MessageConfig implements WebMvcConfigurer {
+    @Bean("messageSource")
+    public MessageSource messageSource() {
+        var messageSource = new ResourceBundleMessageSource();
+
+        // This will match the resources folder files that begin with 'messages'
+        // e.g. messages_wl.properties, messages_fr.properties
+        messageSource.setBasenames("locales/messages", "locales/common-messages");
+        messageSource.setDefaultEncoding("UTF-8");
+        return messageSource;
+    }
+
+    // The LocaleResolver interface has implementations that determine
+    // the current locale based on the session, cookies,
+    // the Accept-Language header, or a fixed value.
+    @Bean
+    public LocaleResolver localeResolver() {
+        var slr = new SessionLocaleResolver();
+        slr.setDefaultLocale(Locale.ENGLISH);
+        slr.setLocaleAttributeName("current.locale");
```

Comment:
Are you sure you need these 2 attributes? In the previously existing LocaleResolver they dont seem to have these.

Reply:
This is gone now